screens/HomeScreen.js

```diff
@@ -8,25 +8,28 @@ import * as WebBrowser from 'expo-web-browser';
 import { MonoText } from '../components/StyledText';
 import { TextList } from '../components';
 
-const ALL_EXPENSES = [
-  { id: 1, name: 'Buy a book', amount: 20 },
-  { id: 2, name: 'Buy dog food', amount: 5 },
-  { id: 3, name: 'Book a flight ticket', amount: 225 },
-  { id: 4, name: 'Buy speakers', amount: 300 },
-  { id: 5, name: 'Book a concert ticket', amount: 50 },
-  { id: 6, name: 'Buy birthday present', amount: 100 },
-  { id: 7, name: 'Buy boots', amount: 400 },
-];
-
 export default function HomeScreen() {
-  const [expenses, setExpenses] = useState(ALL_EXPENSES);
+  const [expenses, setExpenses] = useState([]);
```

Comment:
it would be a better practice to have all "useState" definitions first in a component and different methods afterwards.

so:

```
const [expenses, setExpenses] = useState([]);
const [name, setName] = useState('');
const [amount, setAmount] = useState('');
...
```

then a developer can see all state-related things in one place, which helps to understand a whole component.